packages/components/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,152 @@
+/** @jsx withSlots */
+import * as React from 'react';
```

Comment:
I'm a bit confused... It looks like these are all new files, but you added them before, correct?

Perhaps you need to pull/merge?

Reply:
yes, it's been updated now!